### DISCUSSION TITLE
Finally Laravel Implementation

### DISCUSSION THREAD

**Discussion by silicahd** [givanz/VvvebJs#394] [Show and tell]
I finally had a day to make the implementation of the builder in laravel. It works amazing, I will be pledge to this great project. Thank you.

**Comment by oleteacher** [comment]
Care to share your implementation here on GitHub?

**Reply by silicahd** [reply]
Sadly it is for an internal system we use, so i cant share it.  But the implementation is fairly simple. The main issue is that the builder is made to edit html files but in Laravel you can just point to a page and save it the database by parsing out the body, css and script.

**Reply by hayden-t** [reply]
geez, whats the point of saying 'hey i did this' as if you are sharing it but you arent ?

**Reply by silicahd** [reply]
> geez, whats the point of saying 'hey i did this' as if you are sharing it but you arent ?

There are to many bugs to publish it. :(

**Reply by hayden-t** [reply]
Its fine not to publish, i wouldnt either, but you could offer something of use, maybe there are some PR's you can make ?

**Comment by gbourant** [comment]
Hello @silicahd,

I was going to ask if the VvvebJs can be used with different backend and you achieved that, congrats!

Can you tell me what i need to do in order to use a Java backend?

Don't tell me about implementation details just the high level architecture/steps you did to achieve that!

Thanks in advance!

**Comment by karneaud** [comment]
@silicahd Did you have to include the landing theme? Is there anyway to avoid this @givanz

**Comment by silicahd** [comment]
I am actually working on a new version on this and I will publish it for all.
